similar_dev_search/cli/fetch_repos.py

```diff
@@ -0,0 +1,67 @@
+from pathlib import Path
+
+import click
+from joblib import Parallel, delayed
+
+from similar_dev_search.constants import JSONS_PATH, N_JOBS, REPOS_PATH
+from similar_dev_search.services.file_system import FileSystemService, JsonService
+from similar_dev_search.services.git import GitService, GithubService
+from similar_dev_search.services.git import RepositoryProvider
+from similar_dev_search.setup import setup
+
+
+@click.command()
+@click.option('--username', default='scikit-learn', help='Username for start repo')
+@click.option('--reponame', default='scikit-learn', help='Start repo name')
+@click.option('--max_repos', default=3, help='Max count of repos')
+def fetch_repos(username: str, reponame: str, max_repos: int) -> None:
+    print("Searching repositories...")
+    repos_list = list(find_repos(username, reponame))
+    print("Searching done...")
+    repositories_count = FileSystemService.get_file_count(JSONS_PATH)
```

Comment:
it should be an argument to function - and you can use `JSONS_PATH` as default value